packages/kotti-ui/source/kotti-filter/components/FilterRow.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+	<div class="kt-filter-row">
+		<FilterButtonLink
+			:icon="Yoco.Icon.CLOSE"
+			@click="handleRemove(filter.key)"
+		/>
+		<span class="kt-filter-row__prefix">{{ prefix(index) }}</span>
+		<KtFieldSingleSelect
+			class="kt-filter-row__column-select"
+			:options="columnOptions"
+			size="small"
+			:value="filter.key"
+			@input="handleSetKey"
+		/>
+		<KtFieldSingleSelect
+			class="kt-filter-row__operation-select"
+			:options="operationOptions"
+			size="small"
+			:value="filter.operation"
+			@input="handleSetOperation"
+		/>
+		<component
+			:is="valueComponent"
+			class="kt-filter-row__value-field"
+			:options="valueOptions"
+			size="small"
+			:value="filter.value"
+			@input="handleSetValue"
+		/>
+	</div>
+</template>
+
+<script lang="ts">
+import { KtFieldSingleSelect } from '@3yourmind/kotti-ui'
+import { Yoco } from '@3yourmind/yoco'
+import { computed, defineComponent } from '@vue/composition-api'
+
+import { Kotti } from '../../types'
+import { getOperationOptions, getValueComponent } from '../utils'
+
+import FilterButtonLink from './FilterButtonLink.vue'
+
+export default defineComponent<{
+	index: number
+	filter: Kotti.Filter.Filter
+	column: Kotti.Filter.Column
+	columnOptions: Kotti.FieldSingleSelect.Props['options']
+}>({
+	name: 'FilterRow',
+	props: {
```

Comment:
Please deep sort and remove newlines, to align with the rest of the props in the codebase